src/itest/kotlin/nl/info/zac/itest/ZaakRestServiceTest.kt

```diff
@@ -97,9 +116,577 @@ class ZaakRestServiceTest : BehaviorSpec({
         authenticate(username = TEST_USER_1_USERNAME, password = TEST_USER_1_PASSWORD)
     }
 
-    Given("ZAC Docker container is running and zaakafhandelparameters have been created and a logged-in behandelaar") {
+    Given(
+        "ZAC Docker container is running and zaakafhandelparameters have been created and a raadpleger is logged-in"
+    ) {
+        authenticate(username = TEST_RAADPLEGER_1_USERNAME, password = TEST_RAADPLEGER_1_PASSWORD)
+        lateinit var responseBody: String
+        When("zaak types are listed") {
+            val response = itestHttpClient.performGetRequest("$ZAC_API_URI/zaken/zaaktypes")
+            Then("the response should be a 200 HTTP response") {
+                responseBody = response.body.string()
+                logger.info { "Response: $responseBody" }
+                response.code shouldBe HTTP_OK
+            }
+            And("the response body should contain no zaak types") {
+                JSONArray(responseBody) shouldHaveSize 0
+            }
+        }
+    }
+
+    Given(
+        "ZAC Docker container is running and zaakafhandelparameters have been created and a behandelaar is logged-in"
+    ) {
         authenticate(username = TEST_BEHANDELAAR_1_USERNAME, password = TEST_BEHANDELAAR_1_PASSWORD)
         lateinit var responseBody: String
+        When("zaak types are listed") {
+            val response = itestHttpClient.performGetRequest("$ZAC_API_URI/zaken/zaaktypes")
+            Then("the response should be a 200 HTTP response") {
+                responseBody = response.body.string()
+                logger.info { "Response: $responseBody" }
+                response.code shouldBe HTTP_OK
+            }
+            And("the response body should contain the zaak types") {
+                responseBody shouldEqualJsonIgnoringOrderAndExtraneousFields """
+                [
+                  {
+                    "beginGeldigheid": "$DATE_2025_01_01",
+                    "doel": "$ZAAKTYPE_BPMN_TEST_DESCRIPTION",
+                    "identificatie": "$ZAAKTYPE_BPMN_TEST_IDENTIFICATIE",
+                    "informatieobjecttypes": [
+                      "$TEST_INFORMATIE_OBJECT_TYPE_1_UUID",
+                      "$INFORMATIE_OBJECT_TYPE_BIJLAGE_UUID",
+                      "a8dfd5b8-8657-48bf-b624-f962709f6e19",
```

Comment:
Do we really want to test on all this data in the response? Just worried about the maintainability. And we are really only interested in the correct zaaktypes right?